config/config.go

```diff
@@ -898,6 +966,12 @@ func InitConfig() {
 		}
 	}
 
+	// If the config file defines a "Continue" key, we parse all the yaml files in that directory
```

Comment:
Update this comment to say "ContinuedConfigLocation" key or whatever the term is decided to be.